src/components/pages/Modal/ProfileRenderModal.js

```diff
@@ -42,10 +42,10 @@ const ProfileRenderModal = props => {
   };
 
   useEffect(() => {
-    getProfileData(authState).then(res => {
+    getProfileData(user).then(res => {
```

Comment:
Update on the getProfileData function, we'll no longer need to pass the user parameter in since we modified the authState on API/index.js. You can still leave it on line 48 though